ui/components/Layout.tsx

```diff
@@ -61,6 +61,7 @@ const AppContainer = styled.div`
   min-height: 100vh;
   margin: 0 auto;
   padding: 0;
+  overflow-x: hidden;
```

Comment:
:thinking: Not sure about this one. Never scroll horizontally?

Reply:
It's how I keep the filter bar off the screen - when do we want to scroll horizontally?

Reply:
> when do we want to scroll horizontally?

Whenever we need to? As long as there isn't information that we can't get to :shrug: